src/flow_graph.py

```diff
@@ -240,11 +240,13 @@ def normalize_likely_branches(function: Function) -> Function:
     return new_function
 
 
-def prune_unreferenced_labels(function: Function, rodata: Rodata) -> Function:
+def prune_unreferenced_labels(
+    function: Function, data_section: DataSection
```

Comment:
Hm, this naming does feel very misleading to me. It makes me ask "what is it using the .data section for? which .data section?" when the answer is "it passes in all data-y sections from all files merged together"

Reply:
Yeah, it is misleading. Do you think I could solve this with a comment/docstring?